web_monitoring/diffing_server.py

```diff
@@ -48,8 +61,23 @@ def get(self, differ):
         a = query_params.pop('a')
         b = query_params.pop('b')
 
-        # Fetch server response for URLs a and b.
-        res_a, res_b = yield [client.fetch(a), client.fetch(b)]
+        # Special case for local files, for dev/testing.
+        if a.startswith('file://') and b.startswith('file://'):
+            if os.get('WEB_MONITORING_APP_ENV') in ('development', 'testing'):
```

Comment:
Doesn’t this need to be `os.environ.get()`? This doesn’t work for me.

Also, my usual approach would be to make `production` explicit and have the implicit, this-var-is-not-defined result be `development`. Not sure it’s a big deal either way, though.